### PR TITLE
ADD: @config + set default_bank;

### PR DESCRIPTION
**Pull request by wakawakathedev**
Summary:
- Fixes issue where the App would show the "add bank" modal
- adds default_bank in config
- adds `@config` to aliases

Screenshot 2021-09-11 at 11 58 04 PM

### CODE REVIEW

File: src/config/config.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const config = {
+  DEFAULT_BANK: '54.183.16.194',
```
Comment (jamessspanggg) [inline src/config/config.ts:2]: I believe just an `index.ts` file under the `config` folder would be sufficient?

Also, would be gr8 to make the variable name more explicit, i.e. `DEFAULT_BANK_IP_ADDRESS`

File: src/renderer/containers/App.tsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ import {Flip, ToastContainer} from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
 import electronIsDev from 'electron-is-dev';
 
+import {config} from '@config/index'
```
Comment (jamessspanggg) [inline src/renderer/containers/App.tsx:9]: would u still need the `/index` if you're importing from `@config`?

File: src/config/config.ts (hunk 1 above)
Comment (wakawakathedev) [reply src/config/config.ts:2]: did i set it correctly? - tslint is flagging that `@config` has an issue

@jamessspanggg#1464 - idk if i set the config correctly for `@config` alias

File: src/renderer/containers/App.tsx [hunk 3]
```diff
@@ -16,7 +17,7 @@ import {AppDispatch, ProtocolType} from '@renderer/types';
 import {displayErrorToast, displayToast} from '@renderer/utils/toast';
 
 const DEFAULT_BANK = {
-  ip_address: '13.57.215.62',
+  ip_address: config.DEFAULT_BANK,
```
Comment (jamessspanggg) [inline src/renderer/containers/App.tsx:20]: this needs to be updated to `DEFAULT_BANK_IP_ADDRESS` as well

File: src/renderer/containers/Connect/index.tsx [hunk 4]
```diff
@@ -16,7 +17,7 @@ import {displayErrorToast, displayToast} from '@renderer/utils/toast';
 import './Connect.scss';
 
 const initialValues = {
-  ipAddress: '54.183.16.194',
+  ipAddress: config.DEFAULT_BANK,
```
Comment (jamessspanggg) [inline src/renderer/containers/Connect/index.tsx:20]: here too

Comment (wakawakathedev) [reply src/renderer/containers/Connect/index.tsx:20]: soz - also getting comments from @angle943 - updating config + DEFAULT_BANK_IP_ADDRESS

File: src/renderer/containers/App.tsx (hunk 3 above)
Comment (wakawakathedev) [reply src/renderer/containers/App.tsx:20]: soz - also getting comments from @angle943 - updating config + DEFAULT_BANK_IP_ADDRESS

**Review by angle943 — APPROVED** [review]

**Pull request merged**